express-get-json/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "express-get-json",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
+}
```

Comment:
Each Express project should have its own installation of `express`.

Reply:
Should be fixed now - I must have installed while on the wrong branch. 
![image](https://user-images.githubusercontent.com/97073517/177467584-0b1f829b-e70b-4cdd-9275-3184330defa8.png)